config/settings.yml

```diff
@@ -0,0 +1,2 @@
+marmotta:
+  url: 'http://localhost:8983/marmotta'
```

Comment:
I decided to install rails_config. Seems like it could be a useful tool for later.